src/Microsoft.Azure.WebJobs.ServiceBus/WebJobs.ServiceBus.csproj

```diff
@@ -131,6 +131,9 @@
     <Compile Include="..\Microsoft.Azure.WebJobs.Host\Triggers\ITriggerDataArgumentBinding.cs">
       <Link>Triggers\ITriggerDataArgumentBinding.cs</Link>
     </Compile>
+    <Compile Include="..\Microsoft.Azure.WebJobs.Host\TypeUtility.cs">
+      <Link>TypeUtility.cs</Link>
+    </Compile>
```

Comment:
My hope is that when we switch ServiceBus over to the binding rules model, this would go away since the SDK would fold the attribute before handing it to SB.